mccsearch/code/mainProgTemplate.py

```diff
@@ -59,36 +59,36 @@ def main():
     # -------------------------------------------------------------------------------------------------
 
     # let's go!
-    print "\n -------------- Read MERG Data ----------"
+    print("\n -------------- Read MERG Data ----------")
     mergImgs, timeList = mccSearch.readMergData(CEoriDirName)
-    print ("-" * 80)
+    print(("-" * 80))
```

Comment:
Recommendation to change `print(("-" * 80))` to `print("-" * 80)` in eight places.  The double nested parens `(( ))` are not needed and only slow down your reader.  Repeated on lines: 64, 75, 77, 80, 82, 86, 89, 112.